### PR TITLE
Add a declared license mapping for "CDDL-1.1 OR GPL 2.0 only WITH Classpath exception 2.0"

### PR DESCRIPTION
**Pull request by nnobelis**
Found in https://repo1.maven.org/maven2/com/sun/mail/javax.mail/1.6.1/javax.mail-1.6.1.pom

Signed-off-by: Nicolas Nobelis <nicolas.nobelis@bosch.io>

### CODE REVIEW

File: spdx-utils/src/main/resources/declared-license-mapping.yml [hunk 1]
```diff
@@ -111,6 +111,7 @@
 "CDDL or GPLv2 with exceptions": CDDL-1.0 OR GPL-2.0-only WITH Classpath-exception-2.0
 "CDDL v1.0 / GPL v2 dual license": CDDL-1.0 OR GPL-2.0-only
 "CDDL v1.1 / GPL v2 dual license": CDDL-1.0 OR GPL-2.0-only
+"CDDL-1.1 OR GPL 2.0 only WITH Classpath exception 2.0": CDDL-1.1 OR GPL-2.0-only WITH Classpath-exception-2.0
```
Comment (fviernau) [inline spdx-utils/src/main/resources/declared-license-mapping.yml:114]: I am unable to find this string in the pom file mentioned in the commit message.

Comment (fviernau) [inline spdx-utils/src/main/resources/declared-license-mapping.yml:114]: I am unable to find this string in the pom file mentioned in the commit message.
Please link to the correct POM file.

**Review by fviernau — CHANGES_REQUESTED** [review]

File: spdx-utils/src/main/resources/declared-license-mapping.yml (hunk 1 above)
Comment (sschuberth) [reply spdx-utils/src/main/resources/declared-license-mapping.yml:114]: Also, the sorting is wrong (please look at the top of the file for instructions).

File: spdx-utils/src/main/resources/declared-license-mapping.yml [hunk 2]
```diff
@@ -521,3 +519,6 @@
 "zope 2.1": ZPL-2.1
 "zope license": ZPL-2.1
 "zope v2.1": ZPL-2.1
+# Map a declared license string to "NONE" in order to discard it.
```
Comment (sschuberth) [inline spdx-utils/src/main/resources/declared-license-mapping.yml:522]: You need to keep the header at the top. (Mark lines before sorting.)

**Review by sschuberth — CHANGES_REQUESTED** [review]

**Comment by nnobelis** [comment]
License was in fact coming for a Clearly Defined curation and will be fixed upstream in Clearly Defined.

**Pull request closed without merging**